QUEST.tsv

```diff
@@ -1214,24 +1214,24 @@ QUEST_20150317_001214	Modestas knew the these gambles were more than just about
-QUEST_20150317_001233	{memo X}However model Stars san was the answer to the question of whether honor who really is Statue of it class fader to apply only one as consideration for the bet. {nl} and a lot of people I tried to model Stars' in order to get the answer to that question. {nl} However, model Stars Mr. does not have lost the bet even in conditions that appear to the some of the impossible. Therefore, everyone was flow years remained free to solve the question of the image from the model's Stars. {nl}
-QUEST_20150317_001234	Model Stars Mr. died in betting shop Undefeated not lost the bet to the death, but as already mentioned, other bets live a life that is not at all, gain even most caused by multiplying the secret image I have left the world to live a life to share around. {nl}
-QUEST_20150317_001235	{memo X}Ultimately, is not unknown Goddess was placed placed or interesting questions and beautiful statue instead of being go enjoyed betting in the life of betting shop Klaipeda in the middle of the Klaipeda. {nl} {nl}
+QUEST_20150317_001218	The moment Modestas agreed to the proposal, the old lady pointed her finger past him. {nl} Modestas turned around and to his surprise, the boulder he had trouble with was effortlessly placed on the wagon. By the time Modestas looked back forward, the old lady had already disappeared. {nl}
+QUEST_20150317_001219	Because Modestas had no ropes or equipment to secure the boulder onto the wagon, he had to lean the wagon and carefully drive it to Klaipeda. {nl} On his way to Klaipeda, he again met the soldiers who he had earlier bet against. Their commanding officer was now aware of what had happened thus far, and wanted to apologize to Modestas for getting him caught and hurt in his subordinates' affairs. {nl}
+QUEST_20150317_001220	The officer returned Modesta's gold to him, and ordered his soldiers to help him transport the boulder back to Klaipeda. {nl}
+QUEST_20150317_001221	The officer and solders had sturdy rope, so they were able to transport the rock without worry of it falling off and unload it at Klaipeda. They decided to accompany Modestas all the way there. {nl} Modestas was very grateful for the help he got from the soldiers. {nl} Along with the soldiers, Modestas again ran into none other than the Alchemist he lost earlier against.
+QUEST_20150317_001222	To Modestas, a promise was a promise, so he immediately handed over the gold the soldiers returned to him to the Alchemist. {nl} The Alchemist, surprised by how Modestas could do this with no complaints or dissatisfaction, took the gold and conversed with him.
+QUEST_20150317_001223	Not wanting to be left out of the conversation between Modestas and the Alchemist, the soldiers couldn't continue working, joined in, and learned about the Alchemist's past with Modestas. {nl} The Alchemist apologized to Modestas for turning his lead into gold, and promised to gather all the support he could for Modestas at Klaipeda. He then left. {nl}
```

Comment:
the Old lady and Alchemist are two separate characters. 
Major Characters in this story:
Modestas
Old Lady
Sigfried, Modesta's friend
A bunch of soldiers and their officer
A man with a injured horse and a nice wagon
An Alchemist who tricks Modestas with lead into gold
A mountain
A Dievdirby who carves